.golangci.yml

```diff
@@ -97,3 +100,12 @@ issues:
       linters:
         - dupword
         - gosec
+    # exclude field alignment linter in tests
+    - path: _test\.go
+      text: "fieldalignment:"
+      linters:
+        - govet
+    # exclude fieldalignment "pointer bytes" failures
+    - text: "pointer bytes"
```

Comment:
Disabling the `pointer bytes` errors - benchstat didn't show any negligible difference after fixing these lint errors. It still seems good though as we'll validate new data structures to ensure the struct layout is optimized.

Verified with a simple struct where I added:
```
foo        int8
bar        int
baz        int8
```

This reported a lint error as we want.